airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/DbSourceDiscoverUtil.java

```diff
@@ -4,23 +4,66 @@
 
 package io.airbyte.integrations.source.relationaldb;
 
+import static io.airbyte.protocol.models.v0.CatalogHelpers.fieldsToJsonSchema;
+
+import com.fasterxml.jackson.databind.JsonNode;
 import com.google.common.collect.Lists;
 import io.airbyte.protocol.models.CommonField;
 import io.airbyte.protocol.models.Field;
 import io.airbyte.protocol.models.JsonSchemaType;
 import io.airbyte.protocol.models.v0.AirbyteCatalog;
 import io.airbyte.protocol.models.v0.AirbyteStream;
 import io.airbyte.protocol.models.v0.CatalogHelpers;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteCatalog;
+import io.airbyte.protocol.models.v0.ConfiguredAirbyteStream;
 import io.airbyte.protocol.models.v0.SyncMode;
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
 import java.util.Objects;
 import java.util.function.Function;
 import java.util.stream.Collectors;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * Contains utilities and helper classes for discovering schemas in database sources.
+ */
 public class DbSourceDiscoverUtil {
```

Comment:
I'm good with leaving testing of this class to blackbox at the connector level for now. But we should unit test this at some point. 
Also, making these methods static is appropriate here, but I always have run into an issue trying to test static methods.